airflow/operators/dagrun_operator.py

```diff
@@ -67,6 +67,7 @@ def execute(self, context):
             dr = trigger_dag.create_dagrun(
                 run_id=dro.run_id,
                 state=State.RUNNING,
+                execution_date=context['execution_date'],
```

Comment:
This changes the default from func.now() to execution_date;

@aoen @saguziel what do you guys think?

Reply:
We don't use this operator at Airbnb, so take this with a grain of salt, but I could see this potentially breaking some use cases (e.g. retrying this task no longer creates a new dagrun). Also might be worth waiting for 2.0 if it breaks backwards compatibility significantly.

Reply:
Thanks for the review @bolkedebruin @aoen. Releasing it as a part of 2.0 is OK to me.